_layouts/home.html

```diff
@@ -32,15 +32,13 @@ <h3>{{ project.title | escape }}</h3>
   </div>
 
   <div class="talks">
-    <h2>Recorded talks</h2>    
-    <div class="talk">
-      <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/j3cnnfGk_78" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-      <h3>The black dog can code too - DrupalSouth 2018</h3>
-    </div>
-    <div class="talk">
-      <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/kPAsB6iRjhM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-      <h3>Theming Drupal in 2017: A New Hope - DrupalSouth 2017</h3>
-    </div>
+    <h2>Recorded talks test</h2>    
+    {% for talk in site.data.talks %}
+        <div class="talk">
+          <iframe width="560" height="315" src="{{ talk.talkUrl }}" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+          <h3>{{ talk.title | escape }}</h3>
```

Comment:
If you're having to `escape` it each time, the problem might be on the YAML side.

Try using quotes in the YAML for these values.